### PR TITLE
[FIX] pos_restaurant, point_of_sale: fix test 05_tip_screen_tour.js

### PR DESCRIPTION
**Pull request by davidmonnom**
Fix tests checks, the test 05_tip_screen_tour.js was failing because the order of orders can vary, so we need to change the check to prevent error related to the order of the orders.

rb error: 99346

### CODE REVIEW

File: addons/point_of_sale/static/src/utils.js [hunk 1]
```diff
@@ -117,6 +117,10 @@ export function loadImage(url, options = {}) {
  * @param {HTMLElement} el
  */
 export function loadAllImages(el) {
+    if (!el) {
+        return Promise.resolve();
+    }
+
```
Comment (caburj) [inline addons/point_of_sale/static/src/utils.js:123]: I think it's better if we can assure that `el` is not null or undefined.

Comment (caburj) [reply addons/point_of_sale/static/src/utils.js:123]: And btw, is this necessary for the runbot error fix?

Comment (davidmonnom) [reply addons/point_of_sale/static/src/utils.js:123]: Yep
image

Comment (davidmonnom) [reply addons/point_of_sale/static/src/utils.js:123]: I don't really know why sometime it can be undefined

**Comment by caburj** [comment]
robodoo r+

**Pull request closed without merging**